client/plots/profileRadar.js

```diff
@@ -85,20 +87,36 @@ class profileRadar extends profilePlot {
 			data2 = [] //term2
 		for (let { module, term1, term2 } of this.terms) {
 			const iangle = i * this.angle - Math.PI / 2
-			this.addData('term2', iangle, i, data2)
-			this.addData('term1', iangle, i, data)
+			const item = this.terms[i]
+			const percentage1 = this.getPercentage(term1)
+			const percentage2 = this.getPercentage(term2)
+			//arc
+			this.radarG
```

Comment:
can you move this arc counter clockwise by `this.angle/2` to get the intuitive user interaction

Reply:
Done